eva-accession-release/src/main/java/uk/ac/ebi/eva/accession/release/io/AccessionedVariantMongoReader.java

```diff
@@ -201,8 +209,8 @@ private VariantSourceEntry buildVariantSourceEntry(String study, String sequence
         return sourceEntry;
     }
 
-    private String buildId(long rs) {
-        return RS_PREFIX + Objects.toString(rs);
+    protected String buildId(long rs) {
```

Comment:
Rename to `buildRsId` or similar if the variant ID construction is extracted to a separate method.